### DISCUSSION TITLE
Setting up with Twilio?

### DISCUSSION THREAD

**Discussion by Therealjosephchrzempiec** [taligentx/dscKeybusInterface#112] [dscKeybusInterface Discussion]
Hello i wonder if it is possible to setup the keypad interface to Twilio to get Alerted when someone breaks in?

Joseph

**Comment by taligentx** [comment]
Sure, there's a Twilio-SMS example.

**Comment by Therealjosephchrzempiec** [comment]
Hello Tligentx thank you. It looks like it sends sms when it changes status? From what I'm reading.

**Comment by taligentx** [comment]
Correct, it's an example meant for you to customize to your needs to get notifications on the topics you'd like.  Feel free to re-open if you run across an issue with the library or example.

**Comment by Therealjosephchrzempiec** [comment]
Thank you Taligentx.
